zebrad/src/commands/start.rs

```diff
@@ -204,6 +207,7 @@ impl StartCmd {
         pin!(mempool_queue_checker_task_handle);
         pin!(tx_gossip_task_handle);
         pin!(progress_task_handle);
+        pin!(rpc_task_handle);
```

Comment:
I had this before however when i added it then i get the panics in several acceptance tests:

```
Opening Zcash network protocol listener 127.0.0.1:58666 failed: Os { code: 98, kind: AddrInUse, message: "Address already in use" }. Hint: Check if another zebrad or zcashd process is running. Try changing the network listen_addr in the Zebra config.
```

This might be fixed by other suggestions so i added it back in https://github.com/ZcashFoundation/zebra/pull/3589/commits/d170366bad5a01e1b0075c3841582271d99aecf7

Reply:
That's a weird error when we're not changing anything about the Zcash listener port.

But it could be related to blocking the tokio executor.